### DISCUSSION TITLE
Welcome to DropPoint Discussions!

### DISCUSSION THREAD

**Discussion by GameGodS3** [GameGodS3/DropPoint#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, introduce yourself and say hi to the community below 😊✌.

**Comment by arabianq** [comment]
Hello. Is there a way to change "Shift + Caps" hotkey to something else?

**Reply by dairiki-ren** [reply]
I don't think there currently is, but this appearently will soon be introduced in v1.3.x. Stay tuned!
